src/functional/ffront/field_operator_ast.py

```diff
@@ -170,7 +170,7 @@ class FunctionDefinition(LocatedNode, SymbolTableTrait):
     id: Coerced[SymbolName]  # noqa: A003  # shadowing a python builtin
     params: list[DataSymbol]
     body: list[Stmt]
-    captured_vars: list[Symbol]
+    closure_symbols: list[Symbol]
```

Comment:
```suggestion
    closure_vars: list[Symbol]
```
As mentioned in-person the naming in the foast is not with respect to what the field contains, but with respect to what it represents.

Reply:
I renamed it.